docs/modules/user-guide/partials/proc_adding-kubernetes-component-to-a-devfile.adoc

```diff
@@ -12,116 +12,45 @@ This section describes how to add a `kubernetes` component to a devfile. A compl
 
 .Procedure
 
-. Define a component using the type `kubernetes`.
+. Define a component using the type `kubernetes` or `openshift`.
 
-. The content can be provided through the `reference` attribute, which points to the file with the component content.
+. The content can be provided through the `uri` or `inlined` property.
 +
-.Adding `kubernetes` component using the `reference` attribute
+.Adding openshift component using the 'uri' property
```

Comment:
There's an inconsistency between line 2 and line 19 in how articles are applied to nouns. There's a similar issue with formatting code elements as monospace (using backticks). The 'uri' property might need backticks instead of vertical ones.